pom.xml

```diff
@@ -71,6 +75,7 @@
                 <configuration>
                     <finalName>${project.artifactId}</finalName>
                     <appendAssemblyId>false</appendAssemblyId>
+                    <attach>false</attach>
```

Comment:
I suggest we instead remove the `appendAssemblyId` above and update the Dockerfile with the new filename: `k3a-lag-exporter-jar-with-dependencies.jar`

Reply:
Done.

Reply:
Ah, overlooked the "the same name as before" part.

Reply:
Like this, then?

Reply:
Yes, lovely! ❤️